client/src/components/App.jsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { FaBeer } from 'react-icons/fa';
+import { FaHeart } from 'react-icons/fa';
+
+
+class App extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+
+    };
+    this.handleClick = this.handleClick.bind(this);
+  }
+
+  handleClick() {
+    console.log('click');
+  }
+
+  render() {
+    return (
+      <div>
+        <div className="parent">
+          <div className="shareAndSaveContainer">
+            <div className="shareButton" onClick={this.handleClick}>
+              <a className="share" href="#"><FaBeer />  Share</a>
+            </div>
+            <div className="saveButton" onClick={this.handleClick}>
+              <a className="save" href="#"><FaHeart />  Save</a>
+            </div>
+          </div>
+
+
+
+          <div className="viewPhotosButton" onClick={this.handleClick}>
+            <a href="https://www.airbnb.com/rooms/3447812/slideshow/?check_in=2019-11-22&check_out=2019-11-25&adults=2&children=0&infants=0">View Photos</a>
```

Comment:
Please remove placeholder link. Also, we don't want to use an `<a>`.

Let's change this to either:

- Update your state object, which can trigger the conditional rendering of the slideshow.
- Use React Router to navigate us to a new page inside the app with `/slideshow` somewhere in the URL

Reply:
I've removed the anchor tag - I don't have everything else in place to adequately update state or route to a new page so I've just removed the tag for now and will implement one of the above solutions when I am at that stage of dev